### ISSUE TITLE
Can't run `sst start` when handler path uses a relative path with `../`

### ISSUE THREAD

**Issue by mittonface** [serverless-stack/serverless-stack#589]
### Problem
I'm trying to reference a handler that lives outside of the directory that is running serverless-stack for _reasons_ ™. My api looks like the following:

```js
    const api = new sst.Api(this, "Api", {
      routes: {
        "ANY /{path+}": {
          function: new sst.Function(this, "ContentApiRouter", {
            runtime: "nodejs12.x",
            handler: "../js/content-api/src/index.koaHandler",
            environment: {
            },
          }),
        },
      },
    });
```

This seems to _deploy_ perfectly fine. But when I run `sst start` I get 
> error: Could not resolve "/Users/brent/Desktop/forestry/teams-serverless/identity/.js"

### Solution
I chased down this error a bit  and it looks like the problem is in
`packages/cli/scripts/start.js:getHandlerFilePath`. 

It's doing a `path.split(".")` which is getting tripped up because of my `../`.  Unless I'm missing some context it shouldn't be too hard to make this function a little more intelligent!

**Comment by mittonface** [comment]
Oh I see this is a duplicate of #583.

**Comment by fwang** [comment]
Hey @mittonface, we should be able to cut a release with the fix by tmr (Friday). I will keep you posted!

**Comment by fwang** [comment]
Fixed in [v0.35.2](https://github.com/serverless-stack/serverless-stack/releases/tag/v0.35.2)
